doc/source/whatsnew/v1.1.0.rst

```diff
@@ -966,6 +966,7 @@ MultiIndex
         left.intersection(right, sort=False)
 
 - Bug when joining 2 Multi-indexes, without specifying level with different columns. Return-indexers parameter is ignored. (:issue:`34074`)
+- Bug in :meth:`DataFrame.stack`, :meth:`DataFrame.unstack`, :meth:`Series.unstack` where the level involved is not always sorted (:issue:`21675`)
```

Comment:
Since this PR is not actually changing behavior I don't think it needs a release note

Reply:
Okay let me remove it